casaos-fix-docker-api-version/run.sh

```
@@ -317,8 +317,8 @@ check_containerd_version() {
     return 1
   fi
   
-  # Get installed version
-  local version=$(dpkg -l | grep containerd.io | awk '{print $3}' | head -n1)
+  # Get installed version (use timeout to prevent hanging if dpkg is locked)
+  local version=$(timeout 10 dpkg -l 2>/dev/null | grep containerd.io | awk '{print $3}' | head -n1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)